src/Components/Products/Products.css

```diff
@@ -0,0 +1,233 @@
+.main-container{
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    align-items: stretch;
+    margin: 0;
+}
+
+/* aside */
+.side-content{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 16%;
+    background-color: var(--lg-color);
+    margin: 1rem;
+}
+.main-content{
+    width: 80%;
+}
+.side-title{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+.filter-btn{
+    background-color: transparent;
+    border-radius: 0;
+    font-weight: bolder;
+    color: var(--font--secondary-color);
+    padding: 0;
+}
+.filter-btn:hover{
+    background-color: transparent;
+}
+.clear-btn{
+    background-color: transparent;
+    color: var(--secondary-hover);
+    border-radius: 0;
+    padding: 0;
+}
+.clear-btn:hover{
+    background-color: transparent;
+}
+.filter-btn:hover,.clear-btn:hover{
+    border-bottom: 2px solid var(--primary-color);;
+}
+.filter-category{
+    padding: 0.8rem;
+    font-weight: bold;
+    font-size: 0.9rem;
+}
+.options{
+    padding: 0.1rem 0.4rem;
+    font-size: 0.9rem;
+}
+.options label{
+    display: block;
+}
+.options label input{
+    margin: 0 0.3rem;
+}
+/* main */
+
+/* product card */
+.grid-3-col{
+    margin: 2rem 1rem;
+    display: grid;
+    /* grid-template-columns: repeat(3,1fr); */
```

Comment:
Please remove the commented code.

Reply:
I may use it in future for responsiveness.So kept it commented for now.Will remove later if not required